Test/Framework/Texture2DNonVisualTest.cs

```diff
@@ -77,6 +78,14 @@ public void FromStreamShouldFailTest(string filename)
                 Assert.Throws<InvalidOperationException>(() => _texture = Texture2D.FromStream(_game.GraphicsDevice, reader.BaseStream));
             }
         }
+
+        [Test]
+        public void FromStreamArgumentNullTest()
+        {
+            Assert.Throws<ArgumentNullException>(() => Texture2D.FromStream(_game.GraphicsDevice, null));
```

Comment:
If you change this to `(Stream)null` it should build.

@dellis1972 - Do we need these weird overloads to `Texture2D.FromStream`?

  https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Graphics/Texture2D.OpenGL.cs#L371
  https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Graphics/Texture2D.OpenGL.cs#L377
  https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Graphics/Texture2D.OpenGL.cs#L412

?